config/webpack.config.js

```diff
@@ -56,9 +56,6 @@ const imageInlineSizeLimit = parseInt(process.env.IMAGE_INLINE_SIZE_LIMIT || '10
 // Check if TypeScript is setup
 const useTypeScript = fs.existsSync(paths.appTsConfig);
 
-// Check if Tailwind config exists
-const useTailwind = fs.existsSync(path.join(paths.appPath, 'tailwind.config.js'));
```

Comment:
we don't need tailwind right?

Reply:
what's a typical use-case for tailwind?

Reply:
tailwindcss is just a toolkit you use when you want to get some fancy classes in CSS that help you build nice UI. Since we use Material UI there is no need for anything like this. Previous FE people seemed to copy pasted some ultra generic webpack config that just slow us down with the extra "crap" in it.